docs/reference/flag-definitions.md

```diff
@@ -4,6 +4,12 @@ description: flagd flag definition
 
 # Flag Definitions
 
+## Metadata
+
+Metadata can be defined at both the flag set (as a sibling of [flags](#flags)) and within each flag.
+Flag metadata conveys arbitrary information about the flag or flag set, such as a version number, or the business unit that is responsible for the flag.
+When flagd resolves flags, the returned [flag metadata](https://openfeature.dev/specification/types/#flag-metadata) is a merged representation of the metadata defined in the flag set, and the metadata defined in the flag, with the metadata defined in the flag taking priority.
```

Comment:
Consider adding a link to the playground.

flagd.dev/playground/?scenario-name=Flag+metadata